src/graphql/query/flssLoansQuery.graphql

```diff
@@ -1,10 +1,10 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
+  $filterObject: [FundraisingLoanSearchFilterInput!]
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
-  $limit: Int = 20
+  $limit: Int = 0
```

Comment:
I could see this biting someone in the future if they don't set a limit in their component - "why is my query returning 0 loans???". We probably want to keep this default limit as 10-20 here, which can then be overridden in the component.

Reply:
thanks for 👀 !